### ISSUE TITLE
Accessibility: Narrator: When focus is on dropdown, its role is not announced

### ISSUE THREAD

**Issue by gaurav42** [OfficeDev/office-ui-fabric-react#5110]
### Bug Report

- __Browser and OS versions__: Windows 10 RS3
- __Microsoft Alias__: gdesh

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)
No

Requested priority: (Blocking, High, Normal, Low) 
Normal

Products/sites affected: (if applicable)
VS marketplace

#### Describe the issue:
When narrator is enabled and keyboard focus is on dropdown, it is not announcing the role of the dropdown control

#### Actual behavior:
Narrator doesn't announce role as 'dropdown' or 'listbox' 

#### Expected behavior:
Narrator should announce role as 'dropdown' or 'listbox'

**Comment by betrue-final-final** [comment]
@michaelangotti and @lynamemi this is confirmed. I thought there was a big kerfuffle about what role this should be from before, so it's odd that it doesn't have one announced. Shouldn't this naturally have a role as semantic html?

**Comment by betrue-final-final** [comment]
We should check in with them. Do you have a contact @aditima ?

**Comment by cliffkoh** [comment]
Closing in favor of #5354.
